learntools/embeddings/nbs/2-exercises.ipynb

```diff
@@ -732,6 +736,12 @@
    "source": [
     "`#q2.solution()`\n",
     "\n",
+    "\n",
+    "#### DB: I think we should aim for as little text as possible in the exercises.  \n",
+    "\n",
+    "#### DB: I've watched people using other MOOCs before, and they tend look for the parts that they do... their eyes skip over any text that isn't either specific instructions or code (and in some cases, they seem to skip over that as well.  In that sense, the words are in the way.\n",
```

Comment:
FWIW, in case it wasn't clear, this cell is intended to be a mockup of the text the user would see if they uncommented and ran `#q2.solution()`. Not that that erases the issue you're raising, but at least it's less text for the user to be intimidated by up front.